### ISSUE TITLE
Issues running repo

### ISSUE THREAD

**Issue by mcecil** [ClarkCGA/multi-temporal-crop-classification-baseline#2]
Issues I found.

1. The main issue if getting the data imported from HuggingFace or MLHub. I think ideally we should have clear instructions for all steps (including installing Large File Storage if needed). Ideally this should all be commands that can be typed in (no point and click). For me, it took a while to figure out how to install lfs, and then I still ran into below issue. 
<img width="987" alt="image" src="https://github.com/ClarkCGA/multi-temporal-crop-classification-baseline/assets/6217921/f3e4a2b3-b065-4891-92fd-6576a3db626c">

2. Config.yaml      We should make it clear where to find the different options in this (i.e. in specific scripts). Sam also mentioned we might need to make updates for this. 

3. Update README with new repo name. In the 'clone' command of Step 1.

4. Remove test code near end of main notebook. (Sam noted this). 

I think these are the main issues. I also mentioned to Sam it might help to have examples for the commands in the README, but I'm not sure if that's typically done. (i.e. examples with specific file paths instead of <file_path>.

**Comment by samKhallaghi** [comment]
Thanks Mike,
Points 2-4 are taken care of. For the first issue, we will update the readme in Huggingface on how to download the data properly.

**Comment by mcecil** [comment]
I think the 'cd' command should be at the beginning of Step 2 (before building Docker)

![image](https://github.com/ClarkCGA/multi-temporal-crop-classification-baseline/assets/6217921/0698ded0-9a83-4065-9f83-b4abae4efe1f)

Installing gitlfs and using the `tar zxvf` command seemed to work.

There are lots of messages (below), but the files seemed to unpack.

![image](https://github.com/ClarkCGA/multi-temporal-crop-classification-baseline/assets/6217921/e310238b-14ac-4a6c-8ef9-f7ac7f3f8844)

I launched the Docker and was able to view the files. Here is the final command I used.

`docker run --gpus all -it -p 8888:8888 -v /home/ubuntu/mike/multi-temporal-crop-classification-baseline:/home/workdir -v /home/ubuntu/mike/multi-temporal-crop-classification-baseline/multi-temporal-crop-classification:/home/data semseg_baseline:v1`
